packages/google-pay-integration/src/google-pay-button-strategy.ts

```diff
@@ -68,6 +69,20 @@ export default class GooglePayButtonStrategy implements CheckoutButtonStrategy {
         const { buyNowInitializeOptions, currencyCode, buttonColor, buttonType, onError } =
             googlePayOptions;
 
+        let state = this._paymentIntegrationService.getState();
+        let paymentMethod: PaymentMethod<GooglePayInitializationData>;
+
+        try {
+            paymentMethod = state.getPaymentMethodOrThrow(this._getMethodOrThrow());
+        } catch (_e) {
+            state = await this._paymentIntegrationService.loadPaymentMethod(
+                this._getMethodOrThrow(),
+            );
+            paymentMethod = state.getPaymentMethodOrThrow(this._getMethodOrThrow());
+        }
+
+        this._countryCode = paymentMethod.initializationData?.storeCountry;
```

Comment:
I think in this case it is better to get this variable from `this. _countryCode`. Because in otherwise we pass it twice in two different places. I'm only talking about this strategy